apps/web/pages/api/book/confirm.ts

```diff
@@ -217,7 +219,11 @@ async function patchHandler(req: NextApiRequest) {
         metadata.entryPoints = results[0].createdEvent?.entryPoints;
       }
       try {
-        await sendScheduledEmails({ ...evt, additionalInformation: metadata });
+        await sendScheduledEmails({
+          ...evt,
+          additionalInformation: metadata,
+          requiresConfirmation: evt.requiresConfirmation,
```

Comment:
```suggestion
```

This line is redundant since it's already declared in `evt`

Reply:
You're right - Was on autopilot thank you :)